services/wfm/src/main/java/org/openkilda/wfm/topology/TopologyConfig.java

```diff
@@ -17,7 +17,10 @@
     private Integer discoveryInterval;
     private Integer discoveryTimeout;
     private Integer discoveryLimit;
-    private float discoverySpeakerFailureTimeout;
+    private final float discoverySpeakerFailureTimeout;
```

Comment:
There's no need to use float type when working with time ("long" fits the most cases). Moreover it may lead to undesirable issues when compare float values.